src/pages/app/navigation-bar/control-button.ts

```diff
@@ -1,14 +1,23 @@
 import h from "hyperscript"
+import cx from "classnames"
 
 type ControlButtonProps = {
-  label: string
+  label: Node | string
   image: string
-}
-export const ControlButton = ({ label, image }: ControlButtonProps) => {
+  className?: string
+} & Partial<HTMLButtonElement>
```

Comment:
making button behave as a `HTMLButtonElement` and expect the same attributes from it